assets/base.css

```diff
@@ -3118,13 +3118,27 @@ details-disclosure > details {
 }
 
 /* Fixed background */
-.animate--fixed {
+.animate--fixed,
+.animate--zoom-in {
   clip-path: inset(0);
 }
 
-.animate--fixed>img:not(.zoom):not(.deferred-media__poster-button),
-.animate--fixed>svg:not(.zoom):not(.deferred-media__poster-button) {
+.animate--fixed > img:not(.zoom):not(.deferred-media__poster-button),
+.animate--fixed > svg:not(.zoom):not(.deferred-media__poster-button),
+.animate--zoom-in > img:not(.zoom):not(.deferred-media__poster-button),
+.animate--zoom-in > svg:not(.zoom):not(.deferred-media__poster-button) {
```

Comment:
I wonder if instead of repeating the same thing for a different class name I should name the new class `animate--fixed-zoom-in` (since the image is fixed as well) and use a more general selector like `div[class*='animate--fixed'` 🤔